### PR TITLE
Recap branch

### PR DESCRIPTION
**Pull request by RCode-9000**
I created a past HackAIs page and created a link to it in the main HackAI page

### CODE REVIEW

File: recap.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        </header>
+        <div class="content" role="main">
+            <div class="content-box description">
+                <h2>
+                    Past HackAIs
+                </h2>
+
+                <img src="resources/Hack%20AI%20Logo.png" style="width:200px; padding-bottom: 20px"/>
+
+                <h3> <center> <u> HackAI 2022 Leaderboard </u> </center> </h3>
```
Comment (zhur123) [inline recap.html:58]: Instead of leaderboard, I would suggest podium, awards, or winning teams

File: recap.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+                <p class="HackAI 2022 1st Place Winners">
+                    
+                    <b>Team Name:</b> Alvin and the Chipmunks <br />
+                    <br />
+                    <b>Members:</b> Mihir Joshi, Jagteshwar Singh, and Alvin Ishimwe <br />
+                    <br />
+                    <b>Project Name:</b> DermAI <br />
+                    <br />
+                    <b>Project Description:</b> "DermAI is an AI-based tool to assist physicians in making skin cancer diagnoses. At DermAI, our goal is to provide the highest accuracy prediction of common skin cancers to aid clinicians in making difficult decisions. Even if a clinician is not a dermatologist, they can use this app in their clinic and get an "extra set of eyes" on a potential cancer diagnosis. We believe this will be beneficial in healthcare settings as misdiagnosis is the leading cause of skin cancer-related deaths" - The DermAI Team<br />
```
Comment (zhur123) [inline recap.html:74]: We can just take their description. I don't think we need quotes. Same for the other teams below

File: recap.html [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                    
+                    <b>Team Name:</b> Brutus 2 <br />
+                    <br />
+                    <b>Members:</b> Abdul Habroune <br />
+                    <br />
+                    <b>Project Name:</b> Automated Attendance using Facial Recognition <br />
+                    <br />
+                    <b>Project Description:</b> "Track attendance and time of attendance using facial recognition" - The Brutus 2<br />
+                    <br />
+                    <b>Project Demonstration:</b> Unavailable <br />
```
Comment (zhur123) [inline recap.html:106]: We can just remove if we don't have anything for them

**Review by zhur123 — COMMENTED** [review]
Small wording suggestions. We may need to change format structure when we add more years so it looks distinct

**Review by EthanHarp — APPROVED** [review]

**Pull request merged**